agent_build_refactored/linux/deb_or_rpm/files/init.d/scalyr-agent-2

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This is a direct copy of the file https://github.com/scalyr/scalyr-agent-2/blob/master/agent_build/linux/deb_or_rpm/files/init.d/scalyr-agent-2